src/main.ts

```diff
@@ -93,9 +119,24 @@ const command: Command = {
 		});
 	},
 	run(helper: Helper, args: TestArgs) {
+		/* istanbul ignore next */
+		function unhandledRejection(reason: any) {
```

Comment:
Do we want to be adding a case for these errors? I believe that node is making them fatal exit codes in an upcoming release.